pkg/persistence/db.go

```diff
@@ -178,6 +175,27 @@ func (c *conn) NewKeyClaim(region, originator, hashID string) (string, error) {
 	return "", err
 }
 
+func generateOneTimeCode() (string, error) {
+	charsets := [2][]rune{[]rune("AEFHJKLQRSUWXYZ"), []rune("2456789")}
+
+	seg1, _ := rand.Int(rand.Reader, big.NewInt(int64(len(charsets))))
+	seg2, _ := rand.Int(rand.Reader, big.NewInt(int64(len(charsets))))
+	seg3, _ := rand.Int(rand.Reader, big.NewInt(int64(len(charsets))))
+
+	oneTimeCode := genRandom(charsets[seg1.Int64()], 3) + genRandom(charsets[seg2.Int64()], 3) + genRandom(charsets[seg3.Int64()], 4)
```

Comment:
Wouldn't removing that option reduce our max key space? What benefits would we gain?

Reply:
Noting that doing any form of math on a saturday is a bad idea, my thought behind this was that an all num-pattern would be the weakest option in this pattern, and as such, while it might reduce the overall keyspace, it raises the floor so to speak.

I've popped the question over to CCCS, as this isn't my area of expertise.